tech/tech/domain/entities/products.py

```
@@ -0,0 +1,13 @@
+from datetime import datetime
+from typing import Optional
+
+class Products:
```

Ruff (0.8.2)

Expected 2 blank lines, found 1

Add missing blank line(s)

(E302)